rfcs/045-amundsen-tour-and-feature-walkthrough.md

```diff
@@ -0,0 +1,94 @@
+- Feature Name: Product Tour
+- Start Date: 2020-01-25
+- RFC PR: [amundsen-io/rfcs#045](https://github.com/amundsen-io/rfcs/pull/45) (after opening the RFC PR, update this with a link to it and update the file name)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Amundsen Product Tour and Feature Walkthrough
+
+## Summary
+
+The Product Tour for Amundsen will be a UI based walkthrough configurable component that will help onboard users into Amundsen. Alternatively, it will help us promote new features added to Amundsen, and educate our users about its use.
+
+## Motivation
+
+Currently, Amundsen has "help tips" icons with tooltips that explain some features. We also have an announcements block in our homepage where we show the latest additions to the product. These can be useful, but they are sometimes inssuficient to help our users to get onboarded or discover new features. We think we can create a better experience to our new users and a quick up to date for existing ones.
+
+The outcome of this feature will be an easier onboarding of new users and an ad-hoc introduction to new features as we add them to Amundsen.
+
+## Guide-level Explanation and UI/UX-level Explanation
+
+For Amundsen users, the Tour will trigger in two different modes. The first is a general "Getting started with Amundsen" walkthough, while the second will highlight different features. Both would be formed by an overlay and a modal that will be attached to elements in the UI.
+
+This modal window will have a "Dimiss" button that would hide the Tour altogether; a "Back" button that would move the user to the previous tour step, a "Next" button that moves it forward and a "Close" button with the usual "X" shape in the top right corner. You can see here [an example](https://react-joyride.com/) of what this tour can do (click on the "Start" button).
+
+For Amundsen maintainers, we will extend the JavaScript configuration file with a block about the tour. This object will have a shape like this:
+
+```js
+...,
+tour: [
+    {
+        path: '/',
+        isFeatureTour: false,
+        isShownOnFirstVisit: true,
+        isShownProgramatically: true,
```

Comment:
could you explain what it does?

Reply:
That would be to include a button to show the tour at any time. This will be good for the overall tours for a page, but not so interesting for feature specific features.